cmd/trustbloc-agent-js-worker/src/trustblocagent.js

```diff
@@ -124,6 +124,9 @@ const TrustBlocAgent = function (opts) {
             createDID: async function (req) {
                 return invoke(aw, pending, this.pkgname, "CreateDID", req, "timeout while creating did")
             },
+            saveDIDToSDS: async function (req) {
```

Comment:
Rename to `saveDID`. The configuration will let the implementation know where to save to.

Reply:
Done!